docs/_index.md

```diff
@@ -11,110 +11,931 @@ sitemap:
-      <a href="tutorials/" class="noanchor"><div>
-        <picture><img src="icons/tutorials.svg" alt="Tutorials" loading="lazy"></picture>
-        <p>Tutorials</p></div>
-      </a>
-    </div>
   </div>
 </div>
+</div>
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="max-page">
+<br>
+<br>
+  <h2 class="frontpage-headers">Go from one machine to thousands</h2>
+  <p>
+    When you connect machines to the cloud you get fleet management tools that let you scale from one prototype to thousands of machines you can manage and operate from one place using the Viam Cloud.
+  </p>
+</div>
+
+{{< tabs class="horizontalheaders platform" navheader="Capabilities">}}
+{{% tab name="Deployment" %}}
+
+<div class="tabcontent">
+
+{{< tabs >}}
+{{% tab name="Fragment" %}}
+
+```json
+// Reusable configuration for using a software package
+{
+  "services": [
+    {
+      "name": "speech-1",
+      "namespace": "viam-labs",
+      "type": "speech",
+      "model": "viam-labs:speech:speechio"
+    }
+  ],
+  "modules": [
+    {
+      "type": "registry",
+      "name": "viam-labs_speech",
+      "module_id": "viam-labs:speech",
+      // specific version to deploy
+      "version": "0.5.2"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="explanation">
+  <div class="explanationtext">
+
+Configure and update hardware, software, and machine learning models for groups of machines in one go.
+Viam has a built-in tool called _{{< glossary_tooltip term_id="fragment" text="fragments" >}}_ for using the same configuration on multiple machines.
+
+[Deploy packages across devices →](/how-tos/deploy-packages/)
+
+  </div>
+</div>
+</div>
+
+{{% /tab %}}
+{{% tab name="Provisioning" %}}
+
+<div class="tabcontent">
+
+{{< tabs >}}
+{{% tab name="Shell" %}}
+
+```sh {class="command-line" data-prompt="$" data-output="3-5,6,7"}
+# Create configuration for provisioning machines with a fragment
+echo "{
+  "manufacturer": "Company",
+  "model": "SmartRover",
+  "fragment_id": "11d1059b-eaed-4ad8-9fd8-d60ad7386aa2"
+}" >> viam-provisioning.json
+
+# Get and run the script to install viam on a board.
+wget https://storage.googleapis.com/packages.viam.com/apps/viam-agent/preinstall.sh
+chmod 755 preinstall.sh
+sudo ./preinstall.sh
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+<div class="explanation">
+  <div class="explanationtext">
+
+Provisioning allows you to complete part of the machine setup during the manufacturing process and perform the rest of the first-time setup once the machine is taken into operation.
+When the machine is taken into operation, it will automatically get the latest configuration and updates.
+
+[Learn about provisioning →](/fleet/provision/)
+
+  </div>
+</div>
+</div>
+
+{{% /tab %}}
+{{% tab name="Observability" %}}
+
+<div class="tabcontent">
+
+{{< tabs >}}
+{{% tab name="Viam App" %}}
+
+{{< imgproc src="/fleet/dashboard.png" alt="Dashboard view of machine status information" resize="1200x" class="imgzoom aligncenter" >}}
```

Comment:
This image-- the black looks a bit weird right up on the top. Would be better centered more vertically. If it's too much to fix that no worries though.

Reply:
Added a bit of margin to the top - Adding more ends up looking weird and then we have to also make the image smaller :/